utopia-remix/app/test-assets/utopia-default-project-main.json

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
does the test need this to be correctly typed? I am afraid that we will touch the PersistentModel and will forget to update this test file – will that break anything?

Reply:
that's a **great** shout @balazsbajorics 
done in https://github.com/concrete-utopia/utopia/pull/5639/commits/594c6e4dcdbfb26b6359e6168baa019c74258eed